config/routes.rb

```diff
@@ -6,17 +6,20 @@
 
   get("/profile/:id", { :controller => "application", :action => "profile" })
 
+  get("profile/care_plans/:id/edit", { :controller => "care_plans", :action => "edit"})
```

Comment:
You could re-write this as

```suggestion
  get "profile/care_plans/:id/edit" => "care_plans"#edit"
```

Reply:
Incredibly, helpful thanks. Was able to keep making progress after making this change.